collaborative-circom/src/lib.rs

```diff
@@ -27,6 +27,15 @@ use serde::Deserialize;
 /// A module for file utility functions.
 pub mod file_utils;
 
+/// An enum representing the ZK proof system to use.
+#[derive(Clone, ValueEnum)]
```

Comment:
Is the derive here as expected? IDK why we did not use if for the MPCPrococol and MPCCurve as well?

Reply:
I use the derive macro now where applicable